airbyte-config/init/src/main/java/io/airbyte/config/init/RemoteDefinitionsProvider.java

```diff
@@ -4,11 +4,8 @@
 
 package io.airbyte.config.init;
 
-import static io.airbyte.config.init.PatchingHelpers.addMissingTombstoneField;
-
-import com.fasterxml.jackson.databind.JsonNode;
 import io.airbyte.commons.json.Jsons;
-import io.airbyte.commons.util.MoreIterators;
+import io.airbyte.config.CombinedConnectorCatalog;
```

Comment:
I don't think I saw this before.

Reply:
Yep it's a sweet addition from @pedroslopez: https://github.com/airbytehq/airbyte/pull/16203